pkg/webhook/hook.go

```diff
@@ -118,7 +143,12 @@ func (h *hook) createWebhook() error {
 			},
 		},
 	}
+
 	err := h.cl.Create(context.TODO(), hook)
+	if k8serrors.IsForbidden(err) {
+		return nil
```

Comment:
Should we log it?

Reply:
no, it's confusing message)
Since our webhook is fully optional